theme/index.hbs

```diff
@@ -289,7 +289,6 @@
 
                 <!-- Apply ARIA attributes after the sidebar and the sidebar toggle button are added to the DOM -->
                 <script>
-                    document.getElementById('sidebar-toggle').setAttribute('aria-expanded', sidebar === 'visible');
```

Comment:
Does this change perhaps belong in https://github.com/rust-lang/mdBook/blob/master/src/theme/index.hbs?